modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -1,6 +1,17 @@
-= Getting Client ID and Client Secret of a registered Client App
+= Getting Client ID and Client Secret of a registered Client Application
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, after you approve a client app to access your API, you can access information about the app from API Manager > Contracts. You can view and reset the unique client ID and client secret for the application. When calling the API, the client app owner needs to pass these credentials if the API is protected with policies.
\ No newline at end of file
+As a client application owner of a registered client, you may be required to provide client application credentials when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information from your contract. 
```

Comment:
"you can access the API using these credentials based on the policies that are applied" -> "you can access the API using these credentials, when policies requiring them are applied"

Reply:
Thanks, Nahuel! But I think "based on the policies" is more clear and more accurate structurally as well. I'd leave it as is. Especially since the example in the next sentence makes it even clearer.

Reply:
I disagree in this. It is not more accurate since only few of the OOTB policies request client application credentials. Most of them do not require them.